### PR TITLE
Advising Base Pages

### PR DESCRIPTION
**Pull request by imanmoreira**
Template pages for the advising workflow.

### CODE REVIEW

**Comment by imanmoreira** [comment]
[Create the Base pages for the Advisor Workflow](https://trello.com/c/FyO9FP9v/170-create-the-base-pages-for-the-advisor-workflow)

File: frontend/src/advising/ManageStudents.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * Convert this React component to a component that's connected to the redux store.
+ * When rendering the connecting component, the props assigned in mapStateToProps, do not need to
+ * be passed down as props from the parent component.
+ */
+export const ManageStudents = connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(withRouter(ManageStudentsComponent));
```
Comment (andrewjleung) [inline frontend/src/advising/ManageStudents.tsx:32]: Do we want to transition to using hooks for Redux as well? [`useSelector`](https://react-redux.js.org/api/hooks#useselector) and [`useDispatch`](https://react-redux.js.org/api/hooks#usedispatch) may be better options here for convenience and readability in the future once these pages are fleshed out.

File: frontend/src/advising/GenericAdvisingTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  margin: 30px;
+  background-color: "#ff76ff";
+`;
+const TabsWrapper = styled.div`
+  margin: 20px -30px;
+`;
+
+const PATH_0 = `/advisor/notifications`;
+const PATH_1 = `/advisor/manageStudents`;
+const PATH_2 = `/advisor/templates`;
```
Comment (andrewjleung) [inline frontend/src/advising/GenericAdvisingTemplate.tsx:26]: Maybe we could store these constants in an array indexed by tab order!

This way to get the start tab number you could do:
```
const startTab = PATHS.findIndex(curPath); // and possibly handle the -1 case with a default tab
```

You can also then reuse the constants within `handleChange`:
```
const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
    history.push(PATHS[newValue]);
    setCurrentTab(newValue);
}
```

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -40,10 +44,25 @@ export const App = ({
...
+const AdvisorRouter = (props: any) => {
+  const { path } = props.match;
+
+  return (
+    <GenericAdvisingTemplateComponent>
+      <Switch>
+        <Route path={`${path}/notifications`} component={NotificationsPage} />
+        <Route path={`${path}/manageStudents`} component={ManageStudents} />
+        <Route path={`${path}/templates`} component={TemplatesPage} />
+      </Switch>
```
Comment (andrewjleung) [inline frontend/src/App.tsx:65]: Should a user be able to navigate to just the '/advisor/` page? Or should this redirect them to a specific tab?

**Review by andrewjleung — APPROVED** [review]
This looks great! Very very exciting!!! Just a few small comments but I've tested it out on the branch and everything seems to be in working order.

File: frontend/src/advising/ManageStudents.tsx (hunk 1 above)
Comment (imanmoreira) [reply frontend/src/advising/ManageStudents.tsx:32]: Bet good idea, I'll switch it over

**Pull request merged**